### ISSUE TITLE
Craft 3.1

### ISSUE THREAD

**Issue by missmatsuko** [sjelfull/craft3-mobiledetect#4]
`composer.json` in the latest released version requires `craftcms/cms: ~3.0.0-beta.23`, so I can't install it with Craft 3.1.

The requirement has already been updated to `craftcms/cms": "^3.0.0-RC1` in master.

Not sure if any other changes are needed, or just a new release with the updated `composer.json`.

**Comment by turnstylerj** [comment]
I am having the same issue. Here is the full error output:

```
Composer output: Package "craftcms/vue-asset" listed for update is not installed. Ignoring.
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

Problem 1
- superbig/craft3-mobiledetect 1.0.1 requires craftcms/cms ~3.0.0-beta.23 -> satisfiable by craftcms/cms[3.0.37, 3.0.36, 3.0.35, 3.0.34, 3.0.33, 3.0.32, 3.0.31, 3.0.30.2, 3.0.30.1, 3.0.30, 3.0.29, 3.0.28, 3.0.27.1, 3.0.27, 3.0.26, 3.0.26.1, 3.0.25, 3.0.24, 3.0.19, 3.0.23.1, 3.0.23, 3.0.0-beta.23, 3.0.0-beta.24, 3.0.0-beta.25, 3.0.0-beta.26, 3.0.0-beta.27, 3.0.0-beta.28, 3.0.0-beta.29, 3.0.0-beta.30, 3.0.0-beta.31, 3.0.0-beta.32, 3.0.0-beta.33, 3.0.0-beta.34, 3.0.0-beta.35, 3.0.0-beta.36, 3.0.0-RC1, 3.0.0-RC2, 3.0.0-RC3, 3.0.0-RC4, 3.0.0-RC5, 3.0.0-RC6, 3.0.0-RC7, 3.0.0-RC7.1, 3.0.0-RC8, 3.0.0-RC9, 3.0.0-RC10, 3.0.0-RC10.1, 3.0.0-RC11, 3.0.0-RC12, 3.0.0-RC13, 3.0.0-RC14, 3.0.0-RC15, 3.0.0-RC16, 3.0.0-RC16.1, 3.0.0-RC17, 3.0.0-RC17.1, 3.0.0, 3.0.0.1, 3.0.0.2, 3.0.1, 3.0.2, 3.0.3, 3.0.3.1, 3.0.4, 3.0.5, 3.0.6, 3.0.7, 3.0.8, 3.0.9, 3.0.10, 3.0.10.1, 3.0.10.2, 3.0.10.3, 3.0.11, 3.0.12, 3.0.13, 3.0.13.1, 3.0.13.2, 3.0.14, 3.0.15, 3.0.16, 3.0.16.1, 3.0.17, 3.0.17.1, 3.0.18, 3.0.20, 3.0.21, 3.0.22] but these conflict with your requirements or minimum-stability.
- superbig/craft3-mobiledetect 1.0.1 requires craftcms/cms ~3.0.0-beta.23 -> satisfiable by craftcms/cms[3.0.37, 3.0.36, 3.0.35, 3.0.34, 3.0.33, 3.0.32, 3.0.31, 3.0.30.2, 3.0.30.1, 3.0.30, 3.0.29, 3.0.28, 3.0.27.1, 3.0.27, 3.0.26, 3.0.26.1, 3.0.25, 3.0.24, 3.0.19, 3.0.23.1, 3.0.23, 3.0.0-beta.23, 3.0.0-beta.24, 3.0.0-beta.25, 3.0.0-beta.26, 3.0.0-beta.27, 3.0.0-beta.28, 3.0.0-beta.29, 3.0.0-beta.30, 3.0.0-beta.31, 3.0.0-beta.32, 3.0.0-beta.33, 3.0.0-beta.34, 3.0.0-beta.35, 3.0.0-beta.36, 3.0.0-RC1, 3.0.0-RC2, 3.0.0-RC3, 3.0.0-RC4, 3.0.0-RC5, 3.0.0-RC6, 3.0.0-RC7, 3.0.0-RC7.1, 3.0.0-RC8, 3.0.0-RC9, 3.0.0-RC10, 3.0.0-RC10.1, 3.0.0-RC11, 3.0.0-RC12, 3.0.0-RC13, 3.0.0-RC14, 3.0.0-RC15, 3.0.0-RC16, 3.0.0-RC16.1, 3.0.0-RC17, 3.0.0-RC17.1, 3.0.0, 3.0.0.1, 3.0.0.2, 3.0.1, 3.0.2, 3.0.3, 3.0.3.1, 3.0.4, 3.0.5, 3.0.6, 3.0.7, 3.0.8, 3.0.9, 3.0.10, 3.0.10.1, 3.0.10.2, 3.0.10.3, 3.0.11, 3.0.12, 3.0.13, 3.0.13.1, 3.0.13.2, 3.0.14, 3.0.15, 3.0.16, 3.0.16.1, 3.0.17, 3.0.17.1, 3.0.18, 3.0.20, 3.0.21, 3.0.22] but these conflict with your requirements or minimum-stability.
- superbig/craft3-mobiledetect 1.0.1 requires craftcms/cms ~3.0.0-beta.23 -> satisfiable by craftcms/cms[3.0.37, 3.0.36, 3.0.35, 3.0.34, 3.0.33, 3.0.32, 3.0.31, 3.0.30.2, 3.0.30.1, 3.0.30, 3.0.29, 3.0.28, 3.0.27.1, 3.0.27, 3.0.26, 3.0.26.1, 3.0.25, 3.0.24, 3.0.19, 3.0.23.1, 3.0.23, 3.0.0-beta.23, 3.0.0-beta.24, 3.0.0-beta.25, 3.0.0-beta.26, 3.0.0-beta.27, 3.0.0-beta.28, 3.0.0-beta.29, 3.0.0-beta.30, 3.0.0-beta.31, 3.0.0-beta.32, 3.0.0-beta.33, 3.0.0-beta.34, 3.0.0-beta.35, 3.0.0-beta.36, 3.0.0-RC1, 3.0.0-RC2, 3.0.0-RC3, 3.0.0-RC4, 3.0.0-RC5, 3.0.0-RC6, 3.0.0-RC7, 3.0.0-RC7.1, 3.0.0-RC8, 3.0.0-RC9, 3.0.0-RC10, 3.0.0-RC10.1, 3.0.0-RC11, 3.0.0-RC12, 3.0.0-RC13, 3.0.0-RC14, 3.0.0-RC15, 3.0.0-RC16, 3.0.0-RC16.1, 3.0.0-RC17, 3.0.0-RC17.1, 3.0.0, 3.0.0.1, 3.0.0.2, 3.0.1, 3.0.2, 3.0.3, 3.0.3.1, 3.0.4, 3.0.5, 3.0.6, 3.0.7, 3.0.8, 3.0.9, 3.0.10, 3.0.10.1, 3.0.10.2, 3.0.10.3, 3.0.11, 3.0.12, 3.0.13, 3.0.13.1, 3.0.13.2, 3.0.14, 3.0.15, 3.0.16, 3.0.16.1, 3.0.17, 3.0.17.1, 3.0.18, 3.0.20, 3.0.21, 3.0.22] but these conflict with your requirements or minimum-stability.
- Installation request for superbig/craft3-mobiledetect 1.0.1 -> satisfiable by superbig/craft3-mobiledetect[1.0.1].

Running update with --no-dev does not mean require-dev is ignored, it just means the packages will not be installed. If dev requirements are blocking the update you have to resolve those problems.
```

**Comment by turnstylerj** [comment]
Looks like this is fixed now, just needs a new release tagged. 🎉 https://github.com/sjelfull/craft3-mobiledetect/commit/4170a6176a33883213e86d5c5acba49b8c3c716a#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780

**Comment by khalwat** [comment]
See here for the details:

https://github.com/craftcms/cms/issues/3645#issuecomment-455367876

TL;DR is that it is already fixed, the repo just needs to be tagged with a new release.
